src/services/item/plugins/invitation/schema.ts

```diff
@@ -62,18 +62,42 @@ export const invite = {
     properties: {
       invitations: {
         type: 'array',
-        items: { $ref: 'https://graasp.org/invitations/#/definitions/partialInvitation' },
+        items: {
+          type: 'object',
+          required: ['email', 'permission'],
+          properties: {
+            email: { type: 'string', format: 'email' },
+            permission: { type: 'string' },
+          },
+        },
       },
     },
     additionalProperties: false,
   },
   response: {
     200: {
-      type: 'array',
-      items: {
-        $ref: 'https://graasp.org/invitations/#/definitions/invitation',
+      type: 'object',
+      properties: {
+        memberships: {
+          type: 'array',
+          items: {
+            $ref: 'https://graasp.org/item-memberships/#/definitions/itemMembership',
+          },
+        },
+        invitations: {
+          type: 'array',
+          items: {
+            $ref: 'https://graasp.org/invitations/#/definitions/invitation',
+          },
+        },
       },
     },
+    '4xx': {
+      $ref: 'https://graasp.org/#/definitions/error',
+    },
+    500: {
```

Comment:
Currently I saw different way to write the HTTP Status Code. What do you think about this one ?
```suggestion
   [StatusCodes.INTERNAL_SERVER_ERROR]: {
```